### ISSUE TITLE
Wish: set Midi loop

### ISSUE THREAD

**Issue by pl-gauthier** [frescobaldi/frescobaldi#1345]
tl;dr: To have a playing range in the midi player. e.g.: To play from midi measure 123 (why not with and end range with looping (It's a wish post right‽)). 

When recompiling a score with a \midi {} block, the midi player gets reset effectively loading the freshly generated midi track (provided it is *not* currently playing). The "position" cursor also gets reset to the beginning of the track. So many time do I have to scroll back to the section I am working on to listen back to that small error I needed to correct. I would love to have a playing range implemented in the player.

I know, midi measure no.75 might not be related in any way to measure 75 in the LilyPond score but that's a completely different situation that is not a problem at all for me.

 As always, thank you so much for this wonderful software. You know I use the "averaging tempo" function 832372d most every day (pop music obliges).

**Comment by fedelibre** [comment]
> When recompiling a score with a \midi {} block, the midi player gets reset effectively loading the freshly generated midi track (provided it is _not_ currently playing). The "position" cursor also gets reset to the beginning of the track. So many time do I have to scroll back to the section I am working on to listen back to that small error I needed to correct. I would love to have a playing range implemented in the player.

See also #71

**Comment by pl-gauthier** [comment]
Oh, sorry about that. Can this post be archived as duplicate?

**Comment by fedelibre** [comment]
I'm reopening it.

#71 is about remembering the last MIDI player start position. Although in a comment I added the idea of a MIDI loop, I think it's good to have a specific issue, as the implementation might be done in two different stages.
